src/main/java/net/openhft/chronicle/queue/impl/single/TSQueueLock.java

```diff
@@ -30,7 +30,10 @@
 
 /**
  * Implements queue lock via TableStore mechanism.
+ *
+ * @deprecated To be removed in .22
  */
+@Deprecated
 public class TSQueueLock extends AbstractTSQueueLock implements QueueLock {
```

Comment:
ah so queueLock is the only thing that uses this?

Reply:
Yup